auth0/src/main/java/com/auth0/android/authentication/storage/SecureCredentialsManager.kt

```diff
@@ -254,12 +255,39 @@ public class SecureCredentialsManager @VisibleForTesting(otherwise = VisibleForT
         scope: String?,
         minTtl: Int,
         parameters: Map<String, String>
+    ): Credentials {
+        return awaitCredentials(scope, minTtl, parameters, false)
+    }
+
+    /**
+     * Tries to obtain the credentials from the Storage. The method will return [Credentials].
+     * If something unexpected happens, then [CredentialsManagerException] exception will be thrown. Some devices are not compatible
+     * at all with the cryptographic implementation and will have [CredentialsManagerException.isDeviceIncompatible] return true.
+     * This is a Coroutine that is exposed only for Kotlin.
+     *
+     * If a LockScreen is setup and [SecureCredentialsManager.requireAuthentication] was called, the user will be asked to authenticate before accessing
+     * the credentials. Your activity must override the [Activity.onActivityResult] method and call
+     * [SecureCredentialsManager.checkAuthenticationResult] with the received values.
+     *
+     * @param scope    the scope to request for the access token. If null is passed, the previous scope will be kept.
+     * @param minTtl   the minimum time in seconds that the access token should last before expiration.
+     * @param parameters additional parameters to send in the request to refresh expired credentials
+     * @param forceRefresh This will avoid returning the existing credentials and retrieves a new one even if valid credentials exist.
```

Comment:
```suggestion
     * @param forceRefresh this will avoid returning the existing credentials and retrieves a new one even if valid credentials exist.
```